### ISSUE TITLE
Countdown breaks with zoom level different than 100%

### ISSUE THREAD

**Issue by henriquecm** [saadeghi/daisyui#50]
After every transition the offset increases a little.

Only the Large text example seems to be working correctly a different zoom levels.

![image](https://user-images.githubusercontent.com/15136037/119194582-afd8aa80-ba59-11eb-93f7-aef5a4cf0d5d.png)

**Comment by saadeghi** [comment]
@henriquecm Honestly, I haven't seen it in different zoom levels before. Thanks for reporting!  
The style and sizes are logically correct and I don't know why this happens. I will test more things to see if there's a way to fix.

**Comment by saadeghi** [comment]
bad news, it's an old bug on chromium 
- https://stackoverflow.com/questions/54147708/difference-in-line-height-fixable-on-zoomed-view
- https://bugs.chromium.org/p/chromium/issues/detail?id=442084

I should probably look for another approach.

**Comment by Celant** [comment]
Hey @saadeghi! Did you manage to find an alternative way to handle this?

**Comment by saadeghi** [comment]
No. The only way would be using a long list of numbers in HTML instead of using `:before`. Something like:
```htmlp
<span class="countdown">
  <span>01</span>
  <span>02</span>
  <span>03</span>
  .
  .
  .
  <span>99</span>
</span>
```
And that is not efficient at all 😕

**Comment by saadeghi** [comment]
This bug is now fixed on webkit 🥳  
Tested on latest versions of Chrome and Safari (MacOS)

**Comment by R3D2** [comment]
Maybe there is a regression because I have this same problem on webkit. Also, it's not even showing on some components. Didn't see that at first, because I mainly use Firefox
![IMG_1543](https://user-images.githubusercontent.com/5547320/172682382-5ee100dc-15b3-4594-9b5a-86e3196912d0.jpg)
![IMG_1542](https://user-images.githubusercontent.com/5547320/172682385-4209d534-f5c8-49b2-b18c-369f2a8f873d.jpg)
.
